airflow/providers/discord/notifications/discord_notifier.py

```diff
@@ -0,0 +1,86 @@
+#
```

Comment:
see same comments in https://github.com/apache/airflow/pull/31207#discussion_r1191154806
and https://github.com/apache/airflow/pull/31207#pullrequestreview-1424921345